spec.bs

```diff
@@ -1593,29 +1589,37 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
             |auctionSignals|, a [=map/clone=] of |browserSignals|, |perBuyerSignals|,
             |directFromSellerSignalsForBuyer|, |perBuyerTimeout|, |expectedCurrency|, |ig|, and
             |auctionStartTime|.
-          1. Set |generatedBid|'s [=generated bid/id=] to |generatedBids|'s [=list/size=].
-          1. [=list/Append=] |generatedBid| to |generatedBids|.
-
-            Issue: Instead of inserting to |generatedBids| in each component auction that runs in
-            parallel, create a strucure InterestGroupAuction that holds data for each auction
-            separately (<a href="https://github.com/WICG/turtledove/issues/1021">WICG/turtledove#1021</a>).
           1. Let |generateBidDuration| be the [=duration from=] |generateBidStartTime| to |settings|'s
             [=environment settings object/current monotonic time=], in milliseconds.
           1. If |perBuyerCumulativeTimeout| is not null, decrement |perBuyerCumulativeTimeout| by
             |generateBidDuration|.
-          1. If |generatedBid|'s [=generated bid/no bid=] is true, [=iteration/continue=].
-          1. If [=query generated bid k-anonymity count=] given |generatedBid| returns false:
+          1. If |generatedBid|'s [=generated bid/no bid=] is true:
+            1. [=Register a bid for forDebuggingOnly reports=] given |generatedBid| and
+               |generatedBidsForDebuggingReports|.
+            1. [=iteration/continue=].
+          1. Let |bidsToScore| be a new [=list=] of [=generated bids=]
+          1. If [=query generated bid k-anonymity count=] given |generatedBid| returns true:
+            1. Let |bidCopy| be a copy of |generatedBid|.

```

Comment:
maybe "copy" -> "clone".
@domfarolino Is there a [=clone=] to link to for struct? I saw lists/sets/maps have one, but didn't find one for [struct ](https://infra.spec.whatwg.org/#structs).

Reply:
Yeah infra doesn't seem to have a definition for this and it probably should. Could you use the word "clone" manually, and then file a bug against the Infra Standard?

Reply:
filed: https://github.com/whatwg/infra/issues/634